projects/kubernetes-sigs/cluster-api/patches/0023-Support-configuring-bottlerocket-admin-container-image.patch

```diff
@@ -0,0 +1,544 @@
+From 4b8c7b3f61ebcfe8bd746dd0fd9d3dcb1200da1d Mon Sep 17 00:00:00 2001
+From: Jiayi Wang <jiayiwang7@yahoo.com>
+Date: Wed, 23 Nov 2022 09:26:28 -0500
+Subject: [PATCH] Support configuring bottlerocket admin container image
```

Comment:
Can you add the patch number 23/23 here and in other patches? I think last 3 don't have it